### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by deadspidervenom** [SilentChaos512/Treasure-Bags#15]
Is it possible to disable only the starter bag?

**Comment by SilentChaos512** [comment]
To stop it from being given to new players on first spawn? Yes, just remove it from the `treasurebags:starting_inventory` loot table with a data pack.

(https://github.com/SilentChaos512/Treasure-Bags/blob/1.20.x/src/generated/resources/data/treasurebags/loot_tables/starting_inventory.json)

**Comment by deadspidervenom** [comment]
would it be possible to suggest making it a config option? I ask as, while it can be done with a datapack it feels silly having 1 small datapack that just disables a single item in a mod.

**Comment by SilentChaos512** [comment]
Yeah, that seems like a good candidate for a config option. And I have learned how to make this possible recently.
